lib/Controller/ConfigController.php

```diff
@@ -131,7 +143,7 @@ public function oauthRedirect(string $code = '', string $state = ''): RedirectRe
 				'grant_type' => 'authorization_code'
 			]);
 			if (isset($result['access_token'])) {
-				$accessToken = $result['access_token'];
+				$accessToken = $this->crypto->encrypt($result['access_token']);
```

Comment:
If you don't go with IAppConfig, here is how to safely encrypt:
```suggestion
				$accessToken = $result['access_token'];
				$accessToken = $accessToken === '' ? '' : $this->crypto->encrypt($accessToken);
```